hw3/03-form.js

```diff
@@ -0,0 +1,75 @@
+const express = require("express");
+const app = express();
+const bodyParser = require("body-parser");
+const path = require("path");
+const port = process.env.PORT || 5001;
+
+// http://localhost:5001/form should return a form with input elements for username, email, and submit button
+
+// http://localhost:5001/submit should return all the data the user entered
+
+app.use(bodyParser.json());
+app.use(bodyParser.urlencoded({ extended: true }));
+
+const formData = {
+  name: "",
+  email: "",
+  message: "n/a",
+  newsletter: "No, thank you",
+};
+
+app.get("/form", (req, res) => {
```

Comment:
I believe app.get is a functionality in Express JS. Can you check and confirm if we are allowed to use it in this assignment as I have submitted my code using Node JS only?

Reply:
The homework file says "Use only Node.js for exercises 01 and 02; this means no third-party modules." All the mentions of using just Node.js leave out Exercise 03, so I took this to mean that we could use third-party modules.